github/orgs.go

```diff
@@ -160,6 +164,9 @@ func (s *OrganizationsService) Get(ctx context.Context, org string) (*Organizati
 		return nil, nil, err
 	}
 
+	// TODO: remove custom Accept header when this API fully launches.
+	req.Header.Set("Accept", mediaTypeMemberAllowedRepoCreationType)
```

Comment:
Please add a `testHeader` call in the appropriate test to help the maintainers find the header when it needs to be removed and to be consistent with the rest of the repo.